partner_socialmedia/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Humanitarian Logistics Organisation e.V. - Stefan Becker
```

Comment:
@funbaker Remove Copyright from all `__init__`.py

Reply:
Just wondering, there are many __init__.py's with copyright notices?

Reply:
@funbaker Only use `License`, Could you please check [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py)

Reply:
Done.